File: Business/eUniversity.Business.ViewModels/Specialization/SpecializationRowViewModel.cs
using System;
using eUniversity.Business.ViewModels.Base;
using eUniversity.Business.ViewModels.Enums;

namespace eUniversity.Business.ViewModels.Specialization
{
    /// <summary>
    /// View model for specialization row
    /// </summary>
    public class SpecializationRowViewModel : IViewModel
    {
        /// <summary>
        /// Gets or sets the identifier.
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// Gets or sets the name.
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// Gets or sets the description.
        /// </summary>
        public string Description { get; set; }

        /// <summary>
        /// Gets or sets the speciality identifier.
        /// </summary>
        public string SpecialityId { get; set; }

        public EntityStatusEnum Status { get; set; }

        public string LastModified { get; set; }
    }
}

File: Web/eUniversity.Web/Scripts/viewModels/Specialization/specializationRowViewModel.js
function SpecializationRowViewModel(serverModel, save) {
    var self = this;
    self.Id = ko.observable();
    self.Name = ko.observable();
    self.Description = ko.observable();
    self.LastModified = ko.observable();
    self.SpecialityId = ko.observable();

    self.IsSelected = ko.observable();

    if (serverModel)
    ko.mapping.fromJS(serverModel, {}, self);
    
    self.Save = function (viewModel) {
        save({ viewModels: [ko.mapping.toJS(viewModel)] });
    };
}